catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/activity/utils/FormulaEditorCategoryListDataInteractionWrapper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.ui.activity.utils;
+
+import android.support.test.espresso.DataInteraction;
+
+import org.catrobat.catroid.uiespresso.util.wrappers.DataInteractionWrapper;
+
+import static android.support.test.espresso.Espresso.onData;
+
+import static org.catrobat.catroid.uiespresso.util.matchers.FormulaEditorCategoryListMatchers.isFormulaEditorCategoryListView;
+import static org.catrobat.catroid.uiespresso.util.matchers.FormulaEditorCategoryListMatchers.withFormulaEditorCategoryItemName;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.instanceOf;
+
+public class FormulaEditorCategoryListDataInteractionWrapper extends DataInteractionWrapper {
+
+	public FormulaEditorCategoryListDataInteractionWrapper(DataInteraction dataInteraction) {
+		super(dataInteraction);
+	}
+
+	public static FormulaEditorCategoryListDataInteractionWrapper onFormulaEditorCategoryItemWithNameAtPosition(String formulaCategoryItem, int position) {
+		return new FormulaEditorCategoryListDataInteractionWrapper(onData(allOf(instanceOf(String.class), withFormulaEditorCategoryItemName(formulaCategoryItem)))
```

Comment:
also to be noted, if this whole class will contain only this one static function, then theres no point in having it in a wrapper class in the first place.
the wrappers are intended to provide a way to perform more complicated actions on a certain object while keeping the espresso "look and feel". please have a look at:
https://confluence.catrob.at/display/CAT/Espresso+testing+-+Using+wrappers+instead+of+utility+functions